### ISSUE TITLE
Icon request: icon-infinity (icon-infinite)

### ISSUE THREAD

**Issue by GabrielDelepine** [FortAwesome/Font-Awesome#1647]
Hi,

I need the infinity sign, for example to include in a sentence (quantity : icon-infinity).

All policies don't have the same symbol so I think include it in the font-awesome projet can be a good idea.

![83033bfa-aaad-11e4-9926-6bab122b891a](https://cloud.githubusercontent.com/assets/1979087/6501397/14c7ab16-c318-11e4-8b95-073cd3bcc87c.png)

![icon-infinity](https://f.cloud.github.com/assets/1979087/855743/6984cb02-f52e-11e2-976b-0b8f7c5eedd6.jpg)

Thanks for this very awesome project !

**Comment by no** [comment]
+1

**Comment by arjitc** [comment]
+1 for this.

**Comment by mktcode** [comment]
![](https://assets-cdn.github.com/images/icons/emoji/unicode/1f44d.png)

**Comment by cvsguimaraes** [comment]
+1

**Comment by kugaevsky** [comment]
:thumbsup:

**Comment by AndrewFreemantle** [comment]
:+1:
I prefer the split infinity sign, but any would be great, thank you! :smile:

![image](https://cloud.githubusercontent.com/assets/1304992/5996629/83033bfa-aaad-11e4-9926-6bab122b891a.png)

**Comment by kill-9-me** [comment]
Come on guys It's been a year and a half. Make this happen already.

**Comment by magoo-pl** [comment]
+1

**Comment by jamazerto** [comment]
+1

**Comment by gbts** [comment]
+1

**Comment by Yappli** [comment]
@AndrewFreemantle : I updated the issue with your proposals.

**Comment by lastonoga** [comment]
+1

**Comment by honzsik** [comment]
+1

**Comment by AndrewFreemantle** [comment]
@Yappli :+1:

**Comment by dimdam** [comment]
+1

**Comment by mhorbul** [comment]
+1

**Comment by TomySpagnoletti** [comment]
:+1:

**Comment by VitalyKovalev** [comment]
:+1:

**Comment by VitalyKovalev** [comment]
For those who can't wait bold Helvetica infinity symbol very close to FA style.

**Comment by mewcrazy** [comment]
+1

**Comment by alexburner** [comment]
Why must you toy with us 
http://fortawesome.github.io/Font-Awesome/icon/skyatlas/

**Comment by rained23** [comment]
2 years still not added, while skyatlas did haha...

**Comment by alexburner** [comment]
So close yet so far...

**Comment by coryvb123** [comment]
:+1:

**Comment by SuperSpe** [comment]
I neeeeeed ittttttttttttttt.. Two years and not trace of inifnity-icon....is there a lobby here? illuminati?

**Comment by skechboy** [comment]
:+1:

**Comment by jhiver** [comment]
+1

**Comment by Thyme1152** [comment]
+1

**Comment by sweettimes** [comment]
+1 (Y)

**Comment by xewl** [comment]
+1 unregardless of there being an html/ascii version for it, the infinity sign is missing from this font since 2013 - ahw!

**Comment by astanley86** [comment]
Looking forward to this icon! +1

**Comment by kkprince** [comment]
:+1::+1::+1:

Really want to use it. Why isn't it included yet?

**Comment by kkprince1998** [comment]
Possible solution for ppl who want it, just add this css:

```css
.fa-infin:before {
  font-family: "Helvetica Neue Light", "HelveticaNeue-Light", "Helvetica Neue", Calibri, Helvetica, Arial !important;
  content: '\221E';
  speak: none;
}
```

However, this uses the browser's font, but it should work just like normal font-awesome

**Comment by cba85** [comment]
:+1:

**Comment by wviana** [comment]
+1 👍🏻

**Comment by TheChen5** [comment]
+1

**Comment by pirafrank** [comment]
+1 
In the referenced issue it is explicitly stated that because of different dimensions, including an infinity icon here would be better.

**Comment by demian2g** [comment]
:+1:

**Comment by marketingguy** [comment]
:+1:

**Comment by sweettimes** [comment]
Here ∞

**Comment by kkprince1998** [comment]
That is not font awesome.

**Comment by manhhaiphp** [comment]
👍

**Comment by SamBrishes** [comment]
+1

**Comment by rained23** [comment]
because its infinity , you need to wait for an infinite of time :hourglass_flowing_sand:

**Comment by kkprince1998** [comment]
ie we will neva get it.

**Comment by towfiqi** [comment]
+1

**Comment by codyn329** [comment]
+1

**Comment by BlueManLine** [comment]
+1 :+1: 

On flaticon.com http://www.flaticon.com/search/infinite there are bunch of svg infinite symbols.

**Comment by Cheater912** [comment]
+1

**Comment by vtigranv** [comment]
+1

**Comment by henocdz** [comment]
:+1:

**Comment by codynguyen1116** [comment]
+1

**Comment by KANekT** [comment]
+1

**Comment by supertassu** [comment]
+1

**Comment by MartinLyne** [comment]
+1

**Comment by luzmcosta** [comment]
+1

**Comment by colinschoen** [comment]
Would be great to have

**Comment by mathiasdb** [comment]
+1

**Comment by kaaai3** [comment]
+1

**Comment by alexburner** [comment]
They've added # and % but still no ∞

Sad days.

**Comment by kkprince1998** [comment]
Why is there no plans to get a ∞? It's taking a ∞ amount of time.

**Comment by 77bishap** [comment]
+1

**Comment by 77bishap** [comment]
+1

**Comment by jacek213** [comment]
+1

**Comment by Hexagon** [comment]
:+1:

**Comment by arnaudleyder** [comment]
+1

**Comment by ralphcrisostomo** [comment]
+1

**Comment by marcoelho97** [comment]
+1

**Comment by soportealumne** [comment]
+1

**Comment by kkprince1998** [comment]
+1

**Comment by kkprince1998** [comment]
(Yes I voted twice)

**Comment by ChristWolves** [comment]
+1

**Comment by PeterShaggyNoble** [comment]
+1

**Comment by Sqrcz** [comment]
+1

**Comment by TakitIsy** [comment]
+1

**Comment by spencero91** [comment]
:+1:

**Comment by patchee500** [comment]
+1

**Comment by lhoezee** [comment]
+1

**Comment by moutonnoireu** [comment]
+1

**Comment by rwsansom** [comment]
+1

**Comment by jackoliver** [comment]
+1

**Comment by gburlet** [comment]
:+1:

**Comment by zhirzh** [comment]
:+1:

_possible use case_: a music player with loop three options: none, one, **all**
or an app that has some sort of limit (reminder delay, etc)

**Comment by paolooo** [comment]
+1

**Comment by erikgrueter1** [comment]
+ 1

**Comment by UltimateBrent** [comment]
+1

**Comment by creativefull** [comment]
+1

**Comment by kkprince1998** [comment]
3 Years and nothing. Wow, didn't know it would be this hard to make.

**Comment by petkovve** [comment]
:thumbsup:

**Comment by jaswsinc** [comment]
👍

**Comment by jbrassard** [comment]
+1

**Comment by ChristianKipke** [comment]
(y)

**Comment by Atomkraftzwerg** [comment]
+1

**Comment by SlainTrue** [comment]
👍

**Comment by overint** [comment]
+1

**Comment by alexburner** [comment]
Happy anniversary, still +1

**Comment by RaXoR** [comment]
:+1:

**Comment by chanjohnathan** [comment]
+100

**Comment by DieterGrunewald** [comment]
👍

**Comment by JustDoItSascha** [comment]
:+1:

**Comment by mizzos** [comment]
:+1:

**Comment by hmundt** [comment]
+1
use case: to indicate infinite validity period in case the approving authority did not specify an expiration date on the issued certificate.

**Comment by iamgaurav** [comment]
👍

**Comment by olefasting** [comment]
+1
Having sizing issues when using the browsers built in font...

**Comment by neshymama** [comment]
👍

**Comment by mlametta** [comment]
+1

**Comment by aanred** [comment]
+1

**Comment by Wasabules** [comment]
+1

**Comment by maldechavda** [comment]
+1

**Comment by deanpeterson** [comment]
![image](https://cloud.githubusercontent.com/assets/1156788/21822074/6ec59266-d73b-11e6-9278-27bed0a1cbe9.png)

**Comment by dhm80** [comment]
+1

**Comment by lyndseypadget** [comment]
+1

**Comment by hasusuf** [comment]
+1

**Comment by Lutacon** [comment]
+1

**Comment by eugenevdm** [comment]
+1

**Comment by fisher377164** [comment]
+1

**Comment by adrianratajczak** [comment]
+1

**Comment by mhenley-uk** [comment]
:+1:

**Comment by marcuslongwell** [comment]
👍

**Comment by Driver86** [comment]
Мда... с 2013 года тянется эта тема, и до сих пор не могут добавить символ бесконечности. Лол

**Comment by alexburner** [comment]
As this point I think its left out just to mess with us

**Comment by healdev** [comment]
+1

I definitely would love to see a nice and artistic infinity sign implemented in the Font-Awesome project would be a much better and flexible alternative to just using HTML...

Thanks!

**Comment by dwyer14** [comment]
👍

**Comment by sweettimes** [comment]
![](https://cloud.githubusercontent.com/assets/1156788/21822074/6ec59266-d73b-11e6-9278-27bed0a1cbe9.png)

**Comment by JasperMulder** [comment]
👍

**Comment by Mitrichlab** [comment]
+1

**Comment by kekwin** [comment]
+1

**Comment by kuinky** [comment]
👍

**Comment by Scukerman** [comment]
+1

**Comment by aturcey** [comment]
:+1:

**Comment by YuvalPruss** [comment]
:+1

**Comment by lokeshpahal** [comment]
:+1:

**Comment by lokeshpahal** [comment]
:+1:

**Comment by sedyshev-a** [comment]
:+1:

**Comment by YuvalPruss** [comment]
👍

**Comment by haslam22** [comment]
👍

**Comment by ryanormrod** [comment]
Obviously here because I was looking for it... :)

**Comment by lotrietb** [comment]
Is this coming any time soon? I cant seem to find it.

**Comment by ferparmur** [comment]
+1

**Comment by replaceits** [comment]
+1

**Comment by jarnovanrhijn** [comment]
+1

**Comment by VicV** [comment]
Jesus christ

**Comment by pwr-reques** [comment]
.fa-infinite:before {
    content: "∞";
    color: #fff;
    font-family: initial;
    font-weight: bold;
    font-size: 64px;
    top: 7px;
    position: relative;
}

**Comment by simonsaysforcepush** [comment]
No, it would not be
content:"∞";

it would be
content:"\221E";

**Comment by wa7eedem** [comment]
why the `copyright ` exist and not the Infinity  😞

**Comment by matt17r** [comment]
+1

**Comment by gehernando** [comment]
+1

**Comment by iredmedia** [comment]
+ 1

**Comment by dotcomly** [comment]
+1

**Comment by savproga** [comment]
+1

**Comment by ahmettmertt** [comment]
really ? +1

**Comment by Incrested** [comment]
+1, and my gosh no response from anyone for so long. Oof.

**Comment by jimmymain** [comment]
damn! this is a surprise. +1

**Comment by My1** [comment]
I think I +1 this too.

**Comment by Thyme1152** [comment]
Font Awesome 5 is released with almost 900 (free) icons, and still no infinity. 🤔 
+1

**Comment by mlametta** [comment]
Most commented icon request doesn't make it into 5, that is unfortunate...

**Comment by premiumcuyuz** [comment]
+1

**Comment by aliksend** [comment]
+1

**Comment by BenMorel** [comment]
+1

**Comment by gam0ny** [comment]
+1

**Comment by SaphiLC** [comment]
wow, and they expect us to purchase v5 when they dont even listen to our needs for .... 5 years? ill stick to the free version until i see an attitude change.

**Comment by YannDanthu** [comment]
👍

**Comment by sensibleworld** [comment]
👋 Hey folks! Wanted to drop in and say that we're getting back on track with icon requests and icon category packs. I'm going to do my best to take care of some of the most requested icons over the coming weeks. Stay tuned!

**Comment by cybervaldez** [comment]
+1

**Comment by sdaverede** [comment]
+1

**Comment by nrmv** [comment]
+1

**Comment by Incrested** [comment]
What the actual heck are the devs smoking? Everyone clearly wants the infinity icon: yet for the past 5 years or so, literally no devs have even said "we'll consider it"

**Comment by My1** [comment]
well someone hasnt been watching what FA has been doing recently:
https://fontawesome.com/community/leaderboard/new
it says in progress, so it's being made.

**Comment by Incrested** [comment]
@My1 even then they held it off until a month ago. Ever since then, basically its been ignored

**Comment by sensibleworld** [comment]
The infinity icon is finished, and will be available when our 5.1 release ships. We're admittedly playing a lot of catch-up on GitHub issues, but are working as hard as we can to get back on track with them. I am literally designing new icons every day.

The leaderboard isn't perfect, and will definitely be improved as we move forward, but it's currently the best way to keep track of our progress on these icons. If I've added "in progress" it means that I am actively designing the icon, or it has already been designed and is waiting to be shipped/released.

**Comment by sensibleworld** [comment]
An (almost) infinite amount of time has passed since this icon was requested. Seemed prudent to finally release it.
